test/attributes_test.rb

```diff
@@ -92,7 +92,7 @@ def test_uniqueness
 
     def test_explicit_attributes
       recipe = Recipe.new
-      assert_equal nil, recipe.title
+      assert_nil recipe.title
```

Comment:
and these were based on this warning:

```
DEPRECATED: Use assert_nil if expecting nil from /Users/sam/src/spyke/test/attributes_test.rb:116. This will fail in Minitest 6.
```